### PR TITLE
Update babel example output of runtime usage

### PR DESCRIPTION
**Pull request by Dashed**

### CODE REVIEW

**Comment by Dashed** [comment]
This is output from `v5.2.13`.

**Comment by Dashed** [comment]
Updated  to `v5.2.15` and re-checked output. No change.

File: docs/usage/runtime.md [hunk 1]
```diff
@@ -169,7 +179,9 @@ the `runtime` transformer however turns this into:
 ```javascript
 "use strict";
 
-var _babelHelpers = require("babel-runtime/helpers");
+var _interopRequireDefault = require("babel-runtime/helpers/interop-require-default").default;
```
Comment (sebmck) [inline docs/usage/runtime.md]: Should be `["default"]`

Comment (Dashed) [inline docs/usage/runtime.md]: Is that the expected output?

This is what's output though in OSX. 

```sh
$ pbpaste | babel --optional runtime
"use strict";

var _interopRequireDefault = require("babel-runtime/helpers/interop-require-default").default;

var _bar = require("bar");

var _bar2 = _interopRequireDefault(_bar);

$ babel --version
5.2.16
```

Comment (Dashed) [inline docs/usage/runtime.md]: I'm using `iojs-v2.0.0` via `nvm`. Unsure if that matters.

Comment (thejameskyle) [inline docs/usage/runtime.md]: This should be the output, yours might be reading from a babelrc or something, but the [es3.memberExpressionLiterals transformer](http://babeljs.io/docs/advanced/transformers/es3/member-expression-literals/) should be transforming `.default` to `['default']`.
```sh
➜  babel.github.io git:(master) echo "import foo from 'bar';" | babel --optional runtime
'use strict';

var _interopRequireDefault = require('babel-runtime/helpers/interop-require-default')['default'];

var _bar = require('bar');

var _bar2 = _interopRequireDefault(_bar);
```

**Comment by Dashed** [comment]
Updated to `babel@5.4.7` from `5.3.0`. I can produced the desired output. Still unsure why there was an output regression.

**Pull request closed without merging**